pyscript.core/tests/integration/test_py_config.py

```diff
@@ -66,6 +66,25 @@ def test_py_config_inline(self):
         )
         assert self.console.log.lines[-1] == "config name: foobar"
 
+    @pytest.mark.skip("ERROR_SCRIPT: works with <py-script> not with <script>")
+    def test_py_config_inline_scriptpy(self):
+        self.pyscript_run(
+            """
+        <py-config>
+            name = "foobar"
+        </py-config>
+
+        <script type="py" async>
+            from pyscript import window, document
+            promise = await document.currentScript._pyodide.promise
```

Comment:
I've honestly no idea why this test pass at all as nowhere in (current?) code we attach `_pyodide` to scripts ...

Reply:
no idea either: I didn't write this test, but I'm happy to rewrite it in a better way in some subsequent PR